kpi/models/asset.py

```diff
@@ -1204,6 +1206,8 @@ def __copy_hidden_fields(self, fields: Optional[list] = None):
           `_deployment_data` has been altered directly
         """
 
+    # FIXME: this model should have a `unique_together` constraint
```

Comment:
mystery comment that i'd made :confused: let's remove it. it probably had to do with `AssetSnapshot`, but that's been sorted out in #4025 (really the previous work in the NLP branch that #4025 is now testing)

Reply:
aaand, @noliveleger removed it already :sparkles: